### PR TITLE
docs: add README

### PR DESCRIPTION
**Pull request by patriciapedrosaa**
## Context 
This pull request aims to provide the documentation that provides information necessary to understand the project's objective.

## Changes 

- [x] Add README

### CODE REVIEW

**Review by pedroyremolo — COMMENTED** [review]
missing some important informations about your project, like:
- How can I execute
- How it will be delivered
- What will be the errors returned by the application
- You've mentioned authentication by token, but it's not explicit which type it is

What do you think to check out these links:
https://github.com/hackergrrl/art-of-readme/blob/master/README-pt-BR.md
https://github.com/stone-payments/stoneco-best-practices/blob/master/readme/README_pt.md

**Comment by pedroyremolo** [comment]
still missing endpoints errors responses, like (but not only):
- 404 when retrieving an account balance
- 500 for possible database connection errors
- 400 for invalid name on account creation

**Review by pedroyremolo — COMMENTED** [review]
nice catch putting a license, but we need some adjustments yet o/

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```
[the remark this answers is not in the document]
Comment (patriciapedrosaa) [reply LICENSE:3]: You're right!  
I will make the adjustments.

File: README.md [hunk 2]
```diff
@@ -1 +1,340 @@
...
+    ```
+
+## Environment variables
+
+The existing environment variables in the application are listed below followed by their descriptions:
+
+| Name                 |  Description              |
+| -------------------- | ------------------------- |
+|  ACCESS_SECRET       | Sign key to generate jwt  |
+|  API_PORT            | port to start application |
```
Comment (pedroyremolo) [inline README.md:248]: missing log level env var here

File: README.md [hunk 3]
```diff
@@ -1 +1,340 @@
...
+```
+
+```bash
+$  make run
+```
+
+```bash
+$  make test
+```
+
```
Comment (pedroyremolo) [inline README.md:298]: missing command for building docker image

**Review by pedroyremolo — COMMENTED** [review]
quite better! but i'm still missing some stuff. Besides the comments left, what do you think to add the success responses when available?

**Review by pedroyremolo — APPROVED** [review]

**Pull request merged**